.eslintrc.json

```diff
@@ -37,7 +47,7 @@
           ["^~", "./src/client"],
           ["^\\$components", "./src/client/scss/components"]
         ],
-        "extensions": [".jsx", ".js", ".scss", ".css"]
+        "extensions": [".jsx", ".js", ".tsx", ".ts", ".scss", ".css"]
```

Comment:
curious to know the context behind this file's line changes. We have a separate eslint for typescript already iirc

Reply:
The addition of the new file extensions allow `.js` and `.jsx` files to detect imported `.ts` and `.tsx` files when their file extensions are not specified in the imports.